http/response_writer.go

```diff
@@ -0,0 +1,33 @@
+package http
+
+import (
+	"net/http"
+)
+
+// InstrumentedResponseWriter is a http ResponseWriter that supports recording the status code and
+// response size.  This is used in logging, metrics, and tracing
+type InstrumentedResponseWriter struct {
+	http.ResponseWriter
+	length     int
+	statusCode int
+}
+
+func (w *InstrumentedResponseWriter) Write(b []byte) (n int, err error) {
+	n, err = w.ResponseWriter.Write(b)
+	w.length += n
+	return
+}
+
+func (w *InstrumentedResponseWriter) WriteHeader(statusCode int) {
+	w.ResponseWriter.WriteHeader(statusCode)
+	w.statusCode = statusCode
+	return
```

Comment:
We don't need it here.

Reply:
I removed that file now, i realized that we already had negroni installed, and it is a much more robust response writer (handling all of the various hijack interfaces, etc)